app/assets/stylesheets/application.scss

```diff
@@ -13,7 +13,7 @@
  *= require_tree .
  *= require_self
  */
-
+ @import "bootstrap";
 
 a:link {
```

Comment:
lets remove these css. we do not need it now as we are using bootstrap

Reply:
okay